src/detect-flow-pkts.c

```diff
@@ -23,114 +23,141 @@
 #include "detect-engine-uint.h"
 #include "detect-parse.h"
 
-static int DetectFlowPktsToClientMatch(
+#define DETECT_FLOW_TO_SERVER 1
```

Comment:
generally the code using `TOCLIENT` and `TOSERVER` (no `_`), shall we do that here too?